src/csvcubed/cli/error_mapping.py

```diff
@@ -75,6 +99,8 @@ def friendly_error_mapping(error: ValidationError) -> str:
         NoMeasuresDefinedError: "At least one measure must be defined in a cube.",
         NoUnitsDefinedError: "At least one unit must be defined in a cube.",
         ObservationValuesMissing: "Observed values missing in '{error.csv_column_title}' on rows: {error.row_numbers}",
+        PivotedObsValColWithoutMeasureError: "An observation value column has been defined without a measure linked within the column definition.",
```

Comment:
Can we have this printing out a list of the observed values columns which don't have measures?